libraries/RC_Channel/RC_Channel.h

```diff
@@ -572,6 +576,9 @@ class RC_Channels {
 
     // Allow override by default at start
     bool _gcs_overrides_enabled = true;
+
+    // true if GCS is performing a RC calibration
+    bool is_calibrating;
```

Comment:
can we clarify this boolean's name?  maybe gcs_in_RC_calibration or something like that.